Content.Shared/Delivery/SharedDeliverySystem.cs

```diff
@@ -210,9 +210,17 @@ protected void UpdateDeliverySpawnerVisuals(EntityUid uid, int contents)
 
     protected virtual void GrantSpesoReward(Entity<DeliveryComponent?> ent) { }
 
+    protected virtual void HandlePenalty(Entity<DeliveryComponent> ent, string? reasonLoc = null) { }
+
     protected virtual void SpawnDeliveries(Entity<DeliverySpawnerComponent?> ent) { }
 }
 
+/// <summary>
+/// Used to gather the multiplier from all different delivery components.
+/// </summary>
+[ByRefEvent]
+public record struct GetDeliveryMultiplierEvent(float Multiplier = 0.0f);
```

Comment:
```suggestion
public record struct GetDeliveryMultiplierEvent(float Multiplier = 1.0f);
```
should default to one, that way you can either modify the value multiplicatively or additively in subscriptions